cli/src/address_lookup_table.rs

```diff
@@ -67,6 +68,14 @@ impl AddressLookupTableSubCommands for App<'_, '_> {
                         .arg(
                             Arg::with_name("authority")
                                 .long("authority")
+                                .value_name("AUTHORITY_PUBKEY")
+                                .takes_value(true)
+                                .validator(is_pubkey)
+                                .help("Lookup table authority [default: the default configured keypair]")
+                        )
+                        .arg(
+                            Arg::with_name("authority_signer")
+                                .long("authority-signer")
```

Comment:
To facilitate the migration process away from signing these, do you want to have this flag hidden and print a message saying it's deprecated as of 1.12 if it's used? Or do you plan to do that once the feature is enabled on all clusters?

Reply:
It's actually ok to use this option after the feature is activated. The bigger issue is if folks try using `--authority` before the feature is activated so I added a warning to that option. We can add `authority-signer` as an alias to `authority` once the feature is activated everywhere

Reply:
Ok great, aliasing later makes sense to me